README.md

```diff
@@ -9,7 +9,7 @@
 
 [freeCodeCamp.org](https://www.freecodecamp.org) is a friendly community where you can learn to code for free. It is run by a [donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org) to help millions of busy adults transition into tech. Our community has already helped more than 10,000 people get their first developer job.
 
-Our full-stack web development and machine learning curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
+Our full-stack web development and machine learning curriculum are completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
```

Comment:
This, to me, suggests that the web development and machine learning curriculum are separate. There *is* a bit of overlap between the two, and my understanding was the curriculum is unified.